test/Microsoft.SqlTools.ServiceLayer.UnitTests/QueryExecution/DataStorage/SaveAsExcelFileStreamWriterHelperTests.cs

```diff
@@ -0,0 +1,220 @@
+﻿using Microsoft.SqlTools.ServiceLayer.QueryExecution.DataStorage;
+using Moq;
+using System;
+using System.IO;
+using System.IO.Compression;
+using System.Text.RegularExpressions;
+using System.Xml;
+using Xunit;
+
+namespace Microsoft.SqlTools.ServiceLayer.UnitTests.QueryExecution.DataStorage
+{
+    public class SaveAsExcelFileStreamWriterHelperTests : IDisposable
+    {
+        private Stream _stream;
+        public SaveAsExcelFileStreamWriterHelperTests()
+        {
+            _stream = new MemoryStream();
+            using (var helper = new SaveAsExcelFileStreamWriterHelper(_stream, true))
+            using (var sheet = helper.AddSheet())
+            {
+                sheet.AddRow();
+                sheet.AddCell("");
+                sheet.AddCell(null);
+                sheet.AddCell("test string");
+                sheet.AddCell(3);
+                sheet.AddCell(3.5);
+                sheet.AddRow();
+                sheet.AddCell(new DateTime(1900, 2, 28));
+                sheet.AddCell(new DateTime(1900, 3, 1));
+                sheet.AddCell(new DateTime(1, 1, 1, 15, 00, 00));
+            }
+        }
+        Regex contentRemoveLinebreakLeadingSpace = new Regex(@"\r?\n\s*");
+        private void ContentMatch(string fileName)
+        {
+            string referencePath = Path.Combine("QueryExecution",
```

Comment:
This is failing in [Travis builds](https://travis-ci.org/Microsoft/sqltoolsservice/jobs/211004702) because of a file not found error. This happens for file lookups since the path isn't the same as when run locally. The best solution is follow the pattern used by other tests with baseline files - `FormatterUnitTestBase.TestLocationDirectory` is a good start. Steps to follow:
- Move your `DataStorage` folder with test files under `Microsoft.SqlTools.ServiceLayer.Test.Common\TestData`
- Get the path to the DataStorage folder like so:
```

        private static string TestLocationDirectory
        {
            get
            {
                return Path.Combine(RunEnvironmentInfo.GetTestDataLocation(), "DataStorage");
            }
        }
```
- Change your code to 
```
string referencePath = Path.Combine(TestLocationDirectory, "SaveAsExcelFileStreamWriterHelperTests", fileName);
```